app/customer.py

```diff
@@ -0,0 +1,16 @@
+from app import car as client_car
+
+
+class Customer:
+    def __init__(self,
+                 name: str,
+                 product_cart: dict,
+                 location: list,
+                 money: [int, float],
+                 car_brand: str = None,
+                 car_fuel_consumption: [int, float] = None) -> None:
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contain string elements and can be None has typehint Optional[list[str]]

add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contain string elements has typehint list[str]